index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Microfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <nav>
+      <img
+        id="menu"
+        src="https://img.icons8.com/external-tal-revivo-regular-tal-revivo/96/null/external-mobile-application-hamburger-menu-setting-interface-basic-regular-tal-revivo.png"
+      />
+    </nav>
+    <img id="i1" src="imgs/i1.png" alt="illustration" />
+    <section id="content-1">
+      <h1>
+        <span class="clr-blue">Hey there. <br />I'm Blesslin <br /></span>
+        <span class="clr-green"> I'm a Software <br />Developer</span>
```

Comment:
Good job working on the mobile 

- [ ] Kindly make sure your website should look exactly like the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48-988&t=0ECBEqbggTjVdj0j-0) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=Your%20website%20should%20look%20exactly%20like%20the%20design%20(e.g.%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.) 😃 

-  the `background image`
- the `icons` color. It should be `#505F79`  
- the `<p>` color. It should be `#172B4D`
- kindly, make sure you added this button
 ![btn](https://user-images.githubusercontent.com/57408419/227179624-54a4c69e-77aa-4a0e-9331-b0c0f4a5a5fa.JPG)
- Also make sure you fixed the sizs of the `menu` button 
![menu](https://user-images.githubusercontent.com/57408419/227180294-37e0577f-1297-4ac8-947f-7357a8f72199.JPG)



> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![mobile01](https://user-images.githubusercontent.com/57408419/227177710-a6bde6c9-b57c-4513-b83e-bf0da21ab2b2.JPG)| |![mobile](https://user-images.githubusercontent.com/57408419/227177829-cb0bdc48-28b8-466c-b49f-732cc6e20195.JPG)| |

Reply:
Hi Sir, @joel-kalema  In my computer the mobile version the code didn't break , may I know which breakpoint px you checked my code ?

Reply:
ohh, Hello @BlesslinJerishR , sorry for being late, but for this question, I used `375px` as it was [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=Minimum%20mobile%20screen%20size%20should%20be%20375px)
to check it well, you may `click` on the button below two times 
![mob](https://user-images.githubusercontent.com/57408419/227266453-766440ee-7e0c-440a-9402-44bb9f89ab88.JPG)